package.json

```
@@ -27,7 +28,7 @@
     "test": "vue-cli-service test:unit",
     "test:watch": "vue-cli-service test:unit --watch",
     "tokens:build": "style-dictionary build --platform scss --config ./styleDictionary.config.js && prettier 'src/scss/variables/*.scss' --write",
-    "tokens:watch": "npm run tokens:build && chokidar  \"src/tokens/*.js\" -c \"npm run tokens:build\""
+    "tokens:watch": "chokidar  \"src/tokens/*.js\" -c \"npm run tokens:build\""
```

As far as I can tell we're not supporting Windows here (notice that `format` uses single quotes) 🤷 , so we could clean this up a bit, too, and use single quotes

```suggestion
    "tokens:watch": "chokidar  'src/tokens/*.js' -c 'npm run tokens:build'"
```